analyzer/src/funTest/kotlin/managers/Extensions.kt

```diff
@@ -25,6 +25,9 @@ import java.io.File
 
 import org.ossreviewtoolkit.analyzer.PackageManager
 import org.ossreviewtoolkit.model.ProjectAnalyzerResult
+import org.ossreviewtoolkit.model.yamlMapper
```

Comment:
Would the `test-utils` project be a better place for this for re-use?

Reply:
Probably, but for now I was only looking into the functional tests and wanted a helper function that does not require an import. But it can be moved later if needed.